DataFormats/HcalRecHit/test/HcalRecHitDump_cfg.py

```diff
@@ -0,0 +1,48 @@
+# Run with something like the following:
+# cmsRun HcalRecHitDump_cfg.py |& grep ^++HBHE++ | sed 's/to -120 ns/to 0 ns/g' > dump.log
+
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('Test')
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10)
+)
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/afs/cern.ch/user/i/igv/CMSSW_8_1_X_2016-09-26-2300/src/25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT/step3.root')
```

Comment:
general preference is to have not dependence on private afs directories in the public release software